PACKAGES.md

```diff
@@ -61,15 +60,14 @@ found. Currently, package delivery is only being done for Python.
 - Documentation: TBD
 
 ### [Stackdriver Error Reporting](https://cloud.google.com/error-reporting/)
-- GAPIC: TBD
-- gRPC: TBD
+- [GAPIC](https://pypi.python.org/pypi/gapic-google-cloud-error-reporting-v1beta1)
+- [gRPC](https://pypi.python.org/pypi/grpc-google-cloud-error-reporting-v1beta1)
 - Documentation: TBD
 
 ### [Stackdriver Logging](https://cloud.google.com/logging/)
-- [GAPIC](https://pypi.python.org/pypi/gapic-google-logging-v2)
-- [gRPC](https://pypi.python.org/pypi/grpc-google-logging-v2)
-- [Documentation](http://pythonhosted.org/gax-google-logging-v2/):
-  For old `gax` namespace... will be updated soon for `gapic` namespace.
+- [GAPIC](https://pypi.python.org/pypi/gapic-google-cloud-logging-v2)
+- [gRPC](https://pypi.python.org/pypi/grpc-google-cloud-logging-v2)
+- [Documentation](http://gapic-google-cloud-logging-v2.readthedocs.io/en/latest/)
```

Comment:
Please link to `http://gapic-google-cloud-logging-v2.readthedocs.io` (without the `latest`) -- I am soon going to configure the link to point to the docs for the 0.90.0 release rather than HEAD.